src/utils.js

```diff
@@ -76,11 +77,16 @@ const Jimp = configureJimp({
 
 const { SEED, GROWING, GROWN } = cropLifeStage
 
-const shopInventoryMap = shopInventory.reduce((acc, item) => {
+const purchasableItemMap = cowShopInventory.reduce((acc, item) => {
   acc[item.id] = item
   return acc
 }, {})
 
+shopInventory.reduce((acc, item) => {
+  acc[item.id] = item
+  return acc
+}, purchasableItemMap)
```

Comment:
It took me a moment to work out what this modified logic is doing. It wasn't immediately obvious to me that `purchasableItemMap` is being built up across these two `reduce` operations, so that suggests to me that it's worth making this logic little clearer. What do you think about something along the lines of this?

```js
const purchasableItemMap = [...cowShopInventory, ...shopInventory].reduce(
  (acc, item) => {
    acc[item.id] = item
    return acc
  },
  {}
)
```

The idea here is to keep all of the data structure logic in one (hopefully straightforward) block.